modules/virt-configuring-secondary-dns-server.adoc

```diff
@@ -38,14 +63,14 @@ NAME       TYPE             CLUSTER-IP     EXTERNAL-IP      PORT(S)          AGE
 dns-lb     LoadBalancer     172.30.27.5    10.46.41.94      53:31829/TCP     5s
 ----
 
-. Edit the `HyperConverged` CR in your default editor by running the following command:
+. Edit the `HyperConverged` CR again:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ oc edit hyperconverged kubevirt-hyperconverged -n {CNVNamespace}
 ----
 
-. Enable the DNS server and monitoring components according to the following example:
+. Point to the DNS server by adding the `kubeSecondaryDNSNameServerIP` value that you previously retrieved to the enterprise DNS server records. For example:
```

Comment:
```suggestion
. Point to the DNS server by adding the external IP address value that you previously retrieved to the `kubeSecondaryDNSNameServerIP` field in the enterprise DNS server records. For example:
```
Just a suggestion, but the wording to me is a bit unclear that you're adding the value you retrieved in step 5.